### PR TITLE
create a github issue template

### PR DESCRIPTION
**Pull request by groob**
This is a first draft to get us going. Looking for feedback!

screen shot 2018-04-06 at 2 45 11 pm

### CODE REVIEW

**Comment by groob** [comment]
I added a `SUPPORT.md` file which is another github specific section and it adds a "Looking for help?" section in the sidebar. 

Example below:

screen shot 2018-04-06 at 2 45 11 pm

**Comment by muffins** [comment]
@groob would you mind adding the reddit on there? www.reddit.com/r/osquery

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+If this is a usage question and not a bug report, consider asking it in one of our community supported channel. 
+
+Slack: https://osquery-slack.herokuapp.com/
+Stack Overflow: https://area51.stackexchange.com/proposals/117450/osquery
+Reddit: https://www.reddit.com/r/osquery/
+
+-->
+
+### What version of `osquery` are you using? What operating system. Be specific.
+You Can run this query to get the necessary details:
```
Comment (muffins) [inline .github/ISSUE_TEMPLATE.md]: nit: `You can ...`

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<!--
+
+Thanks for filing an issue! If this is a question or feature request, just delete
+everything here and write out the request, providing as much context as you can.
+
+If this is a usage question and not a bug report, consider asking it in one of our community supported channel.
```
Comment (muffins) [inline .github/ISSUE_TEMPLATE.md]: nit: `If this is a usage or deployment question ...`

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+-->
+
+### What version of `osquery` are you using? What operating system. Be specific.
+You Can run this query to get the necessary details: 
+```
+osqueryd -S --line 'SELECT oi.version as osquery_version, os.version AS os_version, os.build AS os_build, os.platform AS os_platform FROM os_version AS os JOIN osquery_info as oi;'
+```
+
+### What did you do?
```
Comment (muffins) [inline .github/ISSUE_TEMPLATE.md]: Maybe `What have you already tried?`

**Pull request merged**